telco-examples/edge-clusters/dhcp-less/eib/telco-edge-cluster.yaml

```diff
@@ -5,8 +5,6 @@ image:
   baseImage: SLE-Micro.x86_64-5.5.0-Default-RT-GM.raw
   outputImageName: eibimage-slemicro55rt-telco.raw
 operatingSystem:
-  rawConfiguration:
-    diskSize: 4G
```

Comment:
The bottom line is:
- this line works forcing the image to be resize to 3G
- If I remove the section, the image will be the minimum (1.9G)
- Adding the custom/script/growfs for the metal3 workflow, it will resize the to the maximum disk size at the first boot. 
So my conclusion is to remove the 2 lines in order to get the minimum image size to be transfer by networking

Reply:
Just bear in mind this works by luck only https://github.com/suse-edge/edge-image-builder/issues/294 as currently EIB doesn't calculate the size... but as long as you don't add too much files there it will work. I'd say it would be better if you use a safer value (3G will be plenty I guess)